src/utils/atLeastHaVersion.ts

```diff
@@ -0,0 +1,21 @@
+export const atLeastHaVersion = (
```

Comment:
Export this from `utils/index.ts` and import it from `utils` above to be consistent with the deepKeys export/import.